code/__defines/subsystem-defines.dm

```diff
@@ -60,6 +60,9 @@
 #define CUT_OVERLAY_IN(ovr, time) addtimer(CALLBACK(src, /atom/.proc/cut_overlay, ovr), time, TIMER_STOPPABLE | TIMER_CLIENT_TIME)
 #define ATOM_USING_SSOVERLAY(atom) (atom.our_overlays || atom.priority_overlays)
 
+// -- SSsounds --
+#define PLAYSOUND_IN(source, soundin, vol, vary, extrarange, falloff, is_global, usepressure, environment, required_preference, required_asfx_toggles, frequency, time) addtimer(CALLBACK(src, /atom/.proc/playsound, source, soundin, vol, vary, extrarange, falloff, is_global, usepressure, environment, required_preferences, required_asfx_toggles, frequency), time, TIMER_STOPPABLE | TIMER_CLIENT_TIME)
```

Comment:
wouldn't it be better to have it be `PLAYSOUND_AFTER`? `PLAYSOUND_IN` sounds like its a input rather than a timer.